google/cloud/security/scanner/audit/org_rules_engine.py

```diff
@@ -562,7 +570,7 @@ def __init__(self, rule_name, rule_index, bindings, mode=None):
         """
         self.rule_name = rule_name
         self.rule_index = rule_index
-        self.bindings = self._get_bindings(bindings)
+        self.bindings = [IamPolicyBinding.create_from(b) for b in bindings]
```

Comment:
Can we leave __init__ for variable setting and not executing code? Arguably if it's needed it should be done prior to using the class and then passed in.